.gitlab-ci.yml

```diff
@@ -355,3 +368,12 @@ publish:s3:mender-monitor:
     - if is_final_tag ${MENDER_MONITOR_VERSION}; then
     -   aws s3 cp output/mender-monitor*.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_PRIVATE}/latest/mender-monitor_latest-1_all.deb
     - fi
+
+publish:s3:mender-monitor:manual:
+  when: manual
+  extends: .template:publish:s3:mender-monitor
+
+publish:s3:mender-monitor:automatic:
+  rules:
+    - if: '$PUBLISH_MENDER_DIST_PACKAGES_AUTOMATIC == "true"'
```

Comment:
Just a quick question:
Reading this: https://docs.gitlab.com/ee/ci/jobs/job_control.html#specify-when-jobs-run-with-rules
I now expect this can be done in one job, like:
```
rules:
  - if: "....." == "true"
  - if: "..." != "true"
    when: manual
```

Reply:
That is true, but I think is much better to have the explicit `:manual` `:automatic` name in the job.

I got used to it in `mender-qa` (since you introduced it, I think) and is an easy way to quickly check what is going to be published the moment you trigger the pipeline, so that you don't end up waiting for it and then unexpectedly gets published.

Unless you insist I'll leave it as it is.